flink-runtime/src/main/java/org/apache/flink/runtime/taskexecutor/TaskManagerConfiguration.java

```diff
@@ -115,6 +115,13 @@ public Configuration getConfiguration() {
 		return tmpDirectories;
 	}
 
+	public String getFirstTmpDirectory(){
```

Comment:
We can remove this method, since it became unused when we changed to generating the code into just a string instead of a temporary file.

Reply:
Just now starting to catch up but the use of the temporary file was the only issue I had when using the original PR. Great to see it's already fixed!